typing-practice-react/src/displayStats.js

```diff
@@ -0,0 +1,53 @@
+export function displayStats(userMistakesCount, userKeyTypeCount, seconds) {
```

Comment:
The function doesn't display anything, it returns a string :) Be as precise as possible with naming functions for what they _do_ :)